### PR TITLE
added students list

### PR DESCRIPTION
**Pull request by benjQuinn**

### CODE REVIEW

File: students.txt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+    Ben Quinn,
```
Comment (LydiaCodes22) [inline students.txt]: Yep, accurate name

File: students.txt [hunk 2]
```diff
@@ -0,0 +1,6 @@
...
+    Steven Lomax,
+    Vee Uye,
+    Cora Marcu,
+}
```
Comment (LydiaCodes22) [inline students.txt:6]: I like that you closed the object with a curly bracket as well as opening with one.

**Review by LydiaCodes22 — APPROVED** [review]

File: students.txt (hunk 1 above)
Comment (t0mbogle) [inline students.txt]:
```suggestion
    Ben Quinn: "Branch owner",
```

**Review by t0mbogle — CHANGES_REQUESTED** [review]

File: students.txt [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+    Ben Quinn: "Branch owner",
+    Steven Lomax,
+    Vee Uye,
+    Cora Marcu,
```
Comment (pbeardey) [inline students.txt:5]: Not in alphabetical order - insert to line 3

**Review by pbeardey — CHANGES_REQUESTED** [review]
Suggestion - order the names alphabetically

**Pull request merged**